packages/dev/sharedUiComponents/declaration.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.scss' {
```

Comment:
This file can go into the src directory, and then there is no need to add it to the "include" list